fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Request.kt

```diff
@@ -131,11 +134,39 @@ class Request : Fuel.RequestConvertible {
         return this
     }
 
-    fun sources(source: (Request, URL) -> Iterable<File>): Request {
+    fun dataPart(dataPart: (Request, URL) -> DataPart): Request {
```

Comment:
ah sorry for not being more clear ... I mean this to become `fun source(source: (Request, URL) -> File)`. So it is easy for usage with only one `File`